docs/template_plugin/src/CMakeLists.txt

```diff
@@ -37,8 +37,8 @@ target_link_libraries(${TARGET_NAME} PRIVATE
     interpreter_backend)
 
 # ATTENTION: uncomment to register a plugin in the plugins.xml file
-# ie_register_plugins(MAIN_TARGET ${TARGET_NAME}
-#                     POSSIBLE_PLUGINS ${TARGET_NAME})
+ie_register_plugins(MAIN_TARGET ${TARGET_NAME}
+                    POSSIBLE_PLUGINS ${TARGET_NAME})
```

Comment:
needed?

Reply:
Will be removed. This PR should reproduce some problems in ngraph. When the problem are solved all this things will be removed